db/seeds.rb

```diff
@@ -3,5 +3,7 @@
 puts "Created admin: admin@example.com / password"
 
 puts "\n### Users"
-FactoryBot.create(:user, email: "user1@example.com", password: "password")
-puts "Created user: user1@example.com / password"
+5.times do |u|
```

Comment:
fiy i like doing `1..5.each do |u|`, because that's the way that is starting from 1 :)